### ISSUE TITLE
Using link to outdated version of NUnit Adapter causes tests to not be found

### ISSUE THREAD

**Issue by pckenney** [exercism/xcsharp#84]
In the section for "Create a New Visual Studio" there is a link to install the NUnit Test Adapter, here is the link given: https://visualstudiogallery.msdn.microsoft.com/6ab922d0-21c0-4f06-ab5f-4ecd1fe7175d

I also followed the steps of the Nuget Package Manager for Solution, and grabbed the latest NUnit.

This led to me installing version 2.0 of the Test Adapter but version 3.x of the NUnit framework.  These are not compatible and the symptom is that your tests just don't appear in the test explorer.

You can see an example of someone having this issue here: http://stackoverflow.com/a/35057937/23933

I feel the documentation would be improved by updating the link and offering a short caution to ensure the versions are the same.

Here is the update url for the 3.0 version of the Test Adapter: https://visualstudiogallery.msdn.microsoft.com/0da0f6bd-9bb6-4ae3-87a8-537788622f2d

**Comment by kytrinyx** [comment]
> I feel the documentation would be improved by updating the link and offering a short caution to ensure the versions are the same.

Yeah, that sounds like a good idea.

This is in [docs/INSTALLATION.md](https://github.com/exercism/xcsharp/blob/master/docs/INSTALLATION.md)

**Comment by ErikSchierboom** [comment]
Agreed. As @kytrinyx mentioned, the documentation found is in this repro. Would you be willing to send a PR?

**Comment by pckenney** [comment]
OK I have submitted a pull request.

Problem was a bit more nuanced than I initially though, since the Exercism Template does use version 2.x of the NUnit framework, so it is correct to install 2.0 of the Adapter in that workflow.

I have explained more fully the issue/fix in the PR.

Feedback on how I did this is welcome, it's my first pull request against a public project like this so let me know if I did it wrong.
